Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,44 @@
-function setAlarm() {}
+function remainingTime(totalSeconds) {
+  let timeRemaining = document.getElementById("timeRemaining");
+  let minutes = Math.floor(totalSeconds / 60);
+  let seconds = totalSeconds % 60;
+
+   //timeRemaining.textContent = "Time Remaining  " + minutes + ":" + seconds;
+
+  if (minutes < 10) {
+    timeRemaining.textContent =
+      "Time Remaining  " + "0" + minutes + ":" + seconds;
+  } else if (seconds < 10) {
+    timeRemaining.textContent =
+      "Time Remaining  " + minutes + ":" + "0" + seconds;
+  } else {
+    timeRemaining.textContent = "Time Remaining  " + minutes + ":" + seconds;
+  }
+}
+
+
+function setAlarm() {
```

Comment:
this function works well, but there is one small issue. There is nothing in your code that says that you cant set multiple alarms at once, so I entered 72 and pressed set alarm and the counter started an then I entered another number and a new counter starter and the numbers were flicking back and forth between the two timers. Try and fix this!

Reply:
Dear @eliza-an thanks for the feedback. I will look at this